ledger/api-server-damlonx/reference-v2/BUILD.bazel

```diff
@@ -41,7 +41,7 @@ da_scala_binary(
         "//ledger/ledger-api-common",
         "//ledger/ledger-api-health",
         "//ledger/participant-state",
-        "//ledger/participant-state/kvutils",
+        "//ledger/participant-state-kvutils:kvutils",
```

Comment:
If you're gonna rename the package, I'd suggest also renaming the targets accordingly.

Reply:
Good point -- will do that.